CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 4.4.0
```

Comment:
what happened to 4.3? test release after the repo migration? maybe add one line of description in the changelog?